<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChildDlg</class>
 <widget class="QDialog" name="ChildDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editing data about a child</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="childBox" native="true">
     <widget class="QLineEdit" name="leName">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>40</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>60</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSurName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSurName">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbNote">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>60</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNote">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>100</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbDateBirth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>72</y>
        <width>80</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of birth</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="deDateBirth">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>70</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSurName</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>deDateBirth</tabstop>
  <tabstop>leNote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChildDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChildDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
